src/testHelpers/adjustedMockSystemApi.ts

```diff
@@ -110,46 +137,17 @@ const asset = (assetId: number): Promise<Option<PalletAssetsAssetDetails>> =>
 		);
 		assets.set(100, insufficientAsset);
 
-		const sufficientAssetInfo = {
-			owner: mockSystemApi.registry.createType(
-				'AccountId32',
-				'0x0987654309876543098765430987654309876543098765430987654309876543'
-			),
-			issuer: mockSystemApi.registry.createType(
-				'AccountId32',
-				'0x0987654309876543098765430987654309876543098765430987654309876543'
-			),
-			admin: mockSystemApi.registry.createType(
-				'AccountId32',
-				'0x0987654309876543098765430987654309876543098765430987654309876543'
-			),
-			freezer: mockSystemApi.registry.createType(
-				'AccountId32',
-				'0x0987654309876543098765430987654309876543098765430987654309876543'
-			),
-			supply: mockSystemApi.registry.createType('u128', 100),
-			deposit: mockSystemApi.registry.createType('u128', 100),
-			minBalance: mockSystemApi.registry.createType('u128', 100),
-			isSufficient: mockSystemApi.registry.createType('bool', true),
-			accounts: mockSystemApi.registry.createType('u32', 100),
-			sufficients: mockSystemApi.registry.createType('u32', 100),
-			approvals: mockSystemApi.registry.createType('u32', 100),
-			status: mockSystemApi.registry.createType(
-				'PalletAssetsAssetStatus',
-				'live'
-			),
-		};
 		const sufficientAsset = mockSystemApi.registry.createType(
 			'PalletAssetsAssetDetails',
-			sufficientAssetInfo
+			multiLocationAssetInfo
 		);
 		assets.set(1984, sufficientAsset);
 
 		const maybeAsset = assets.has(assetId) ? assets.get(assetId) : undefined;
 
 		if (maybeAsset) {
 			return new Option(
-				createStatemineRegistry(9430),
+				createStatemineRegistry(9435),
```

Comment:
I know using westend asset hub is temporary. Should we change this name to reflect that in the meantime?

Reply:
Hmm this is a good question. Im down to keep it as statemine, since it technically doesn't matter. 

But also I am kinda torn on this.